points/module.py

```diff
@@ -71,77 +70,55 @@ async def points_get(self, ctx, member: discord.Member = None):
     @points.command(name="leaderboard", aliases=["best"])
     async def points_leaderboard(self, ctx):
         """Points leaderboard"""
-        embed = utils.Discord.create_embed(
-            author=ctx.author,
-            title=_(ctx, "Points 🏆"),
-            description=_(ctx, "Score, descending"),
+        title = _(ctx, "Points 🏆")
```

Comment:
I'd use text-only title here. "Points leaderboard". I know it is not directly part of the PR, but since we're touching the code, we can do that.